CMakeLists.txt

```diff
@@ -27,21 +35,20 @@ set(SOURCES
     ${CLIENT_SOURCE_DIR}/websocketpp_client.cpp
 )
 
-find_package(Boost REQUIRED COMPONENTS random system)
 find_package(nlohmann_json CONFIG REQUIRED)
+find_package(uuid_v4 REQUIRED)
 find_package(OpenSSL REQUIRED)
 find_package(plog CONFIG REQUIRED)
 find_package(websocketpp CONFIG REQUIRED)
 
 add_library(NostrSDK ${SOURCES} ${HEADERS})
 target_link_libraries(NostrSDK PRIVATE
-    Boost::random
-    Boost::system
```

Comment:
Stating for the record that we _may_ need to re-add Boost at a later date.  I believe `websocketpp` uses it as a dependency, but we don't test the `websocketpp` usage directly, as of yet.

Reply:
You are right, websocketpp uses it as a dependency but vcpkg downloads the required parts of boost (already compiled) for websocketpp

But if we directly use boost, vcpkg will have to download all of it fr and compile it from source which took me around 2hrs, and all that just for generating UUIDs... it's a lot of overhead in my opinion.  

If we need it in the future, we'll cross that bridge when we get to it.